draft-dekater-scion-pki.md

```diff
@@ -1394,167 +1394,98 @@ More information can be found on the SCIONLab website and in the {{SCIONLAB}} pa
+Once the voting representatives have verified the TRC data, the Ceremony Administrator computes the DER encoding of the data and the SHA256 hash value of the TRC payload file. The TRC payload file is then shared with the voting representatives via the data exchange device who verify the TRC payload hash value by computing this on their machine and checking it matches the one displayed by the Ceremony Administrator.
 
-Phase 2 successfully concludes once every voting representative confirms that the contents of the TRC payload are correct.
+Phase 2 concludes when all voting representatives confirm that the contents of the TRC payload are correct.
 
 
 ### Phase 3: TRC Signing {#phase3}
 {:numbered="false"}
 
-In Phase 3, each voting representative attaches a signature created with each one of their private voting keys to the TRC (payload file). They do this on their own machine. The purpose of signing a TRC that contains newly introduced public keys with the corresponding private keys is to prove the possession of the private keys.
+Each voting representative attaches a signature created with their private voting keys to the TRC (payload file), using their own machine. This serves to prove possession of the private keys.
 
-Phase 3 concludes after all voting representatives have cast their votes.
+Phase 3 concludes when all voting representatives have attached their signatures to the TRC.
 
 
 ### Phase 4: TRC Validation {#phase4}
 {:numbered="false"}
 
-In Phase 4, all voting representatives share the signed TRC with the Ceremony Administrator. This happens again over the data exchange device, which goes from one voter to the next. Each voting representative copies the TRC payload signed with the voter's private keys from their own machine onto the data exchange device. The last voter returns the device to the Ceremony Administrator, who assembles the final TRC by aggregating the payload data with the votes (signatures) cast by the voting representatives.
+All voting representatives copy the TRC payload signed with their private voting keys to the data exchange device and return this to the Ceremony Administrator. The Ceremony Administrator assembles the final TRC by aggregating the payload data and verifying the signatures based on the certificates exchanged during Phase 1. The signed TRC is then shared with all the voting representatives who must again inspect the signatures.
```

Comment:
I'd mention the data exchange device 

I'd also clarify the role of the administrator: 
```suggestion
All voting representatives copy the TRC payload signed with their private voting keys to the data exchange device and return this to the Ceremony Administrator. The Ceremony Administrator assembles the final TRC by aggregating the payload data and verifying the signatures based on the certificates exchanged during Phase 1. The Ceremony Administrator then shares the assembled TRC with all participants who MUST again inspect the signatures and verify them based on the certificates exchanged in Phase 1.
```